go/util/datetime/date_time.go

```diff
@@ -65,3 +74,16 @@ func (dt *DateTime) UnmarshalNoms(v types.Value) error {
 	*dt = DateTime{time.Unix(int64(s), int64(frac*1e9))}
 	return nil
 }
+
+func RegisterHRSCommenter(tz *time.Location) {
+	types.UnregisterHRSCommenter(datetypename, hrsEncodingName)
```

Comment:
You're right. There's no need. I removed that line and modified the test to check that.